src/lib/pages/backoffice/Series/SeriesDetails/SeriesMultipartMonographs/SeriesMultipartMonographs.js

```diff
@@ -39,7 +45,9 @@ export default class SeriesMultipartMonographs extends Component {
   };
 
   volumeFormatter = ({ row }) => {
-    const { seriesPid, seriesType } = this.props;
+    const { seriesDetails } = this.props;
+    const seriesPid = seriesDetails.metadata.pid;
+    const seriesType = seriesDetails.metadata.mode_of_issuance;
```

Comment:
I have also encountered this, and wanted to ask why do we rename it?

Reply:
no idea what happened there, I just fixed it because it was broken...